tests/queries/0_stateless/01114_database_atomic.sh

```diff
@@ -9,81 +9,98 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
+DATABASE_1="${CLICKHOUSE_DATABASE}_1"
+DATABASE_2="${CLICKHOUSE_DATABASE}_2"
+DATABASE_3="${CLICKHOUSE_DATABASE}_3"
```

Comment:
As I see in the reference file, `CLICKHOUSE_DATABASE` is actually just `default`. It's a bit strange, I would expect it to be `test_{random_str}`. With databases `default_1/2/3` I don't see much difference from `test_01114_1/2/3`

Reply:
Ah, we replace `test_{random_str}` to `default` in the test results. Did't know that